src/mmw/js/src/routes.js

```diff
@@ -12,8 +12,8 @@ router.addRoute(/^/, DrawController, 'draw');
 router.addRoute(/^analyze/, AnalyzeController, 'analyze');
 router.addRoute('project(/:projectId)(/scenario/:scenarioId)(/)', ModelingController, 'project');
 router.addRoute('project/:projectId/clone(/)', ModelingController, 'projectClone');
-//TODO Add route for /project/XX/scenario/compare
-router.addRoute(/^compare/, CompareController, 'compare');
+router.addRoute('project(/:projectId)(/scenario/:scenarioId)(/)', ModelingController, 'project');
```

Comment:
I was just checking, I'm also fixing in my PR, wanted to make sure before removing it.